assets/questions/reverseWords.js

```diff
@@ -126,8 +126,9 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
         'For this question, you\'ll implement the reverseWords function. ',
         'This function takes a string of words and reverses the individual ',
         'words, but it does not change the ordering of the words within the ',
-        'sentence. All the original whitespace/punctuation should be ',
-        'preserved. Here\'s an example:'
+        'sentence. A word is defined to consist of contiguous letters, and ',
+        'all other characters in the string (such as whitespace and ',
+        'punctuation) should be preserved. Here\'s an example:'
```

Comment:
Wording appears a bit awkward to me. How about something like...

A word consists of contiguous letters. Characters that are not letters, such as whitespace and punctuation, should be preserved (not moved or reversed).

Reply:
Thanks, I've updated it -- PTAL. Just FYI, I didn't use this text exactly because it doesn't really make sense to me to "reverse" a character.